pkgs/by-name/ap/apple-sdk/package.nix

```diff
@@ -90,6 +90,11 @@ stdenvNoCC.mkDerivation (
         ln -s "${sdkName}" "$sdkpath/MacOSX${sdkMajor}.sdk"
         ln -s "${sdkName}" "$sdkpath/MacOSX.sdk"
 
+        # Swift adds these locations to its search paths. Avoid spurious warnings by making sure they exist.
+        mkdir -p "$platformPath/Developer/Library/Frameworks"
+        mkdir -p "$platformPath/Developer/Library/PrivateFrameworks"
+        mkdir -p "$platformPath/Developer/usr/lib"
```

Comment:
😔

Will `xcode-select --print-path` print this path?

Reply:
`xcode-select --print-path` prints `/Applications/Xcode.app/Contents/Developer` for me.

These are added by Swift. The current Swift packaging works around the warnings. I opted to fix them in the SDK instead.

Reply:
Oh, I mean with `xcbuild`’s `xcode-select`, in a Nix environment.